tests/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+sudo apt-get update -y
+sudo apt-get install -y bats
+
+# kubectl
+sudo apt-get install -y apt-transport-https ca-certificates curl
+sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+sudo apt-get update -y
+sudo apt-get install -y kubectl
```

Comment:
❓ / ⛏️ Same note here. I would prefer find a way to specify the version of `kubectl` rather than depending on a version available in a distro repository


* https://kubectl.docs.kubernetes.io/installation/kubectl/
* https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/
* https://github.com/bats-core/bats-detik#docker-setup